community/nss/build

```diff
@@ -4,46 +4,35 @@ export NSS_USE_SYSTEM_SQLITE=1
 export NSS_ENABLE_WERROR=0
 export USE_64=1
 export BUILD_OPT=1
+export NSS_DISABLE_DEPRECATED_SEED=1
+export NSS_DISABLE_DEPRECATED_RC2=1
+export NSPR_INCLUDE_DIR=/usr/include/nspr
+
 export CC="${CC:-cc}"
 export CCC="${CXX:-c++}"
 export CXX="${CXX:-c++}"
 
-make -j1 -C nss nss_build_all
+make -C nss all
 
-# TODO: Maybe install the NSS binaries. They're uneeded
-#       right now as we handle certs differently.
-#
-# TODO: Maybe install nss-config. Nothing uses it yet
-#       and this build system is terrible so shhh.
 install -Dt "$1/usr/include/nss" -m644 dist/public/nss/*.h
-install -Dt "$1/usr/include/nss" -m644 dist/Linux*/include/*.h
+install -Dt "$1/usr/include/nss/private" -m644 \
+    dist/private/nss/blapi.h dist/private/nss/alghmac.h
 install -Dt "$1/usr/lib"               dist/Linux*/lib/*.so
 install -Dt "$1/usr/lib"         -m644 dist/Linux*/lib/*.chk
 
 # Yucky. This is needed by qtwebengine.
-install -Dt "$1/usr/include/nss/obsolete" \
-    -m644 dist/Linux*/include/obsolete/*.h
-
-# Install the NSPR files. This is a joint package as I'd
-# rather not juggle which version of NSPR works best with NSS.
-install -Dt "$1/usr/include/nspr" -m644 nspr/pr/include/*.h
-install -Dt "$1/usr/include/nspr" -m644 nspr/lib/*/*.h
+#install -Dt "$1/usr/include/nss/obsolete" \
+#    -m644 dist/Linux*/include/obsolete/*.h
```

Comment:
Probably don't need these commented, but last I checked webengine DID need these headers.

Reply:
I was just able to build successfully without them

Reply:
Then we're safe to remove these lines completely!